deploy/20_cloud-ingress-operator.ClusterRole.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
```

Comment:
do we need to file anymore?

Reply:
Yes. This is still the source of truth for what goes in the SSS and the OLM bundle.